rest-api-spec/src/main/resources/rest-api-spec/api/behavioral_analytics.list.json

```diff
@@ -0,0 +1,41 @@
+{
+  "behavioral_analytics.list": {
+    "documentation": {
+      "url": "http://todo.com/tbd",
+      "description": "Returns the existing behavioral analytics collections."
+    },
+    "stability": "experimental",
+    "visibility": "feature_flag",
+    "feature_flag": "xpack.ent-search.enabled",
+    "headers": {
+      "accept": [
+        "application/json"
+      ],
+      "content_type": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_application/analytics",
+          "methods": [
+            "GET"
+          ]
+        },
+        {
+          "path": "/_application/analytics/{name}",
+          "methods": [
+            "GET"
+          ],
+          "parts":{
+            "name":{
+              "type":"list",
+              "description":"A comma-separated list of analytics collections to limit the returned information"
```

Comment:
nit: is it really needed? I guess it was added for cases where the list is too big? Can we restrict the usage to return all collections? We can limit the number to the soft limit that we have for search (10k), that should be sufficient to handle all deployments? We can also allow a query parameter like we do for Search Application that search the collection by names (wildcards accepted)?

Reply:
So currently we're supporting both the LIST and GET functionality through this one endpoint. Name is optional. If name is specified, it will either return the only analytics collection that matches or return a list that partially matches the name. This is for the search use-case. We modelled this API originally around other ES APIs like cat.  

When I was doing the kibana PR, its evident that this wasn't a great idea. Knowing when its an exact match or a partial match is bad. Im going to propose we split this into two separate endpoints for GET and LIST.